docs/performance/tune-performance.md

```diff
@@ -63,6 +63,7 @@ If an operator called multiple kernels during execution, the performance numbers
 {"cat":"Kernel", "name":<name of the kernel called first>, ...}
 {"cat":"Kernel", "name":<name of the kernel called next>, ...}
 ```
+onnxruntime also offers some [utility](https://github.com/microsoft/onnxruntime/tree/master/tools/perf_view) to render the statistics as a summarized view in the brower.
```

Comment:
"browser" Also, "a" utility instead of "some" since it's just a single tool, right?